carts/views.py

```diff
@@ -11,47 +12,62 @@ class CartsView(View):
-
-            cart, created = Cart.objects.get_or_create(
-                user       = user,
-                product_id = product_id,
-                option_id  = option_id,
-                defaults = {
-                    'quantity' : 0
-                })
-            cart.quantity += quantity
-            cart.save()
-            
+            data         = json.loads(request.body)
+            user         = request.user
+            request_list = data['request_list']
+
+            for request in request_list:
+                product_id = request['product_id']
+                option_id  = request['option_id']
+                quantity   = int(request['quantity'])
+
+                if not Product.objects.filter(id=product_id).exists():
+                    return JsonResponse({"message":"INVALID_PRODUCT_ID"}, status=404)
+
+                if not Option.objects.filter(id=option_id).exists():
+                    return JsonResponse({"message":"INVALID_PRODUCT_ID"}, status=404)
+
+                option = Option.objects.get(id=option_id)
+
+                if option.product_id != int(product_id):
+                    return JsonResponse({"message":"INVALID_OPTION"}, status=404)
+
+                cart, created = Cart.objects.get_or_create(
+                    user       = user,
+                    product_id = product_id,
+                    option_id  = option_id,
+                    defaults   = {
+                        'quantity' : 0
+                    })
+                cart.quantity += quantity
+                cart.save()      
             return JsonResponse({"message":"SUCCESS"}, status=201)
         except KeyError:
             return JsonResponse({"message":"KEY_ERROR"}, status=400)
     
     @login_required
     def get(self, request):
         user = request.user
+
+        if not Cart.objects.filter(user=user).exists():
+            return JsonResponse({"MESSAGE":"EMPTY_CART"}, status=200)
+
         cart_list = [{
-            'id'          : item.id,
-            'product_name': item.product.name,
-            'quantity'    : item.quantity,
-            'size'        : item.option.size,
-            'price'       : item.option.price,
-            'image_url'   : item.product.thumbnail_image_url
+            'id'           : item.id,
+            'product_name' : item.product.name,
+            'quantity'     : item.quantity,
+            'size'         : item.option.size,
+            'product_price': item.option.price,
+            'image_url'    : item.product.thumbnail_image_url,
+            'price'        : int(item.option.price) * int(item.quantity)
         } for item in Cart.objects.filter(user=user)]
-        return JsonResponse({"CART_LIST": cart_list}, status=200)
+        total_price = Cart.objects.filter(user=user).aggregate(total_price=Sum(F('option__price')*F('quantity')))
+
+        shipping_price = 20000
+        if total_price['total_price'] >= 20000 :
+            shipping_price = 0
```

Comment:
```suggestion
        SHIPPING_PRICE = 20000
        
        if total_price['total_price'] >= SHIPPING_PRICE :
            shipping_price = 0
```
일정 금액이 넘었을 때 무료 배송을 하는 것 같은데, 일정 금액이라는 부분은 기획에 따라 바뀔 수 있고, 여러 곳에서 사용할 수도 있으므로 상수화를 해주시면 좋을 것 같습니다. 그리고 기준 금액과 배송료를 구분해서 사용하시면 좋을 것 같아요.